webapp/i18n/en_AU.json

```diff
@@ -305,8 +305,8 @@
   "WelcomePage.NoThanks.Text": "No thanks, I'll figure it out myself",
   "Workspace.editing-board-template": "You're editing a board template.",
   "boardSelector.confirm-link-board": "Link board to channel",
-  "boardSelector.confirm-link-board-button": "Yes, link board",
-  "boardSelector.confirm-link-board-subtext": "Linking the '\\{boardName}'\\ board to this channel would give all members of this channel 'Editor' access to the board. Are you sure you want to link it?",
+  "boardSelector.confirm-link-board-button": "Link board",
+  "boardSelector.confirm-link-board-subtext-with-other-channel": "Linking the \"{boardName}\" board to this channel would give all members of this channel \"Editor\" access to the board.{lineBreak} Additionally, this board is linked to another channel, and will be unlinked from the other channel when you link it here.",
```

Comment:
```suggestion
  "boardSelector.confirm-link-board-subtext-with-other-channel": "When you link "{boardName}" to the channel, all members of the channel (existing and new) will be able to edit it.{lineBreak} This board is currently linked to another channel. It will be unlinked if you choose to link it here.",
```